test/unittests/commands/excel2xml/test_excel2xml_lib.py

```diff
@@ -406,11 +407,14 @@ def test_make_integer_prop(self) -> None:
         self.run_test(prop, method, [int(x) for x in different_values], invalid_values)
 
     def test_make_interval_prop(self) -> None:
-        prop = "interval"
-        method = excel2xml.make_interval_prop
         different_values = ["+.1:+.9", "10:20", "1.5:2.5", "-.1:5", "-10.0:-5.1"]
```

Comment:
Does it make sense that we allow negative values, if we only use it with segment bounds?

Reply:
No, it doesn't make sense. I moved the negative values down to the `self.assertWarns(Warning)` section. In fact, the code remains the same, because we only issue warnings for fishy values, no errors any more. But it's good to assert that this issues a warning, because it doesn't make sense.